tests/installation/bootloader.pm

```diff
@@ -210,7 +210,16 @@ sub run() {
 
     my $args = "";
     if ( get_var("AUTOYAST") ) {
-        $args .= " netconfig=dhcp,all";
+        if (get_var("VERSION")=~/^\D*10\D*/ ) { # sle10 does not have netconfig,ifcfg
+            $args .= " netsetup=dhcp,all";
+        }
+        elsif (get_var("VERSION")=~/^\D*11\D*/ ) { # sle11 does not have ifcfg
+            #$args .= " netconfig=dhcp,all";
+            $args .= " netsetup=dhcp,all";  #hotfix, netconfig seems not to work 
```

Comment:
what's wrong with just using netconfig=dhcp,all? as it was before? it's supported by all